src/lib/maplibre/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as MapLibreUtils } from "./maplibre_utils";
```

Comment:
I think something like `export * from "./utils"` should work. If it doesn't, we have to do something like `export { isPolygon, isLine, isPoint, ... } from "./utils"`. Can generate that list pretty quickly with grep and a regex if you like